src/NLog/Targets/ConsoleTargetHelper.cs

```diff
@@ -41,11 +41,15 @@ internal static class ConsoleTargetHelper
     {
         public static bool IsConsoleAvailable(out string reason)
         {
+            reason = string.Empty;
 #if !SILVERLIGHT && !__IOS__ && !__ANDROID__ && !MONO
             try
             {
                 if (!Environment.UserInteractive)
                 {
+                    if (Type.GetType("Mono.Runtime") != null && Console.In is StreamReader)
```

Comment:
Maybe we should add this to PlatformDetector and create a unit test for it? (we run on Travis which is Mono)